Node/src/TeamsChatConnector.ts

```diff
@@ -53,14 +53,19 @@ export interface IntentMessage extends builder.IMessage {
 }
 
 export class TeamsChatConnector extends builder.ChatConnector {
+  public static queryInvokeName:string = 'composeExtension/query';
+  public static querySettingUrlInvokeName:string = 'composeExtension/querySettingUrl';
+  public static settingInvokeName:string = 'composeExtension/setting';
 
   private allowedTenants: string[];
 
-  private queryHandlers: { [id: string]: ComposeExtensionQueryHandlerType } = {};
+  private queryHandlers: { [id: string]: ComposeExtensionHandlerType } = {};
+  private invokeEventHandlers: { [id: string]: ComposeExtensionHandlerType } = {};
```

Comment:
sure can you rename the queryHandlers to composeExtensionHandlers which will handle all of them?